plugins/authentication/cookie/cookie.php

```diff
@@ -164,8 +168,9 @@ public function onUserAuthenticate($credentials, $options, &$response)
 			 * Delete all tokens for this user!
 			 */
 			$query = $this->db->getQuery(true)
-				->delete('#__user_keys')
-				->where($this->db->quoteName('user_id') . ' = ' . $this->db->quote($results[0]->user_id));
+				->delete($this->db->quoteName('#__user_keys'))
+				->where($this->db->quoteName('user_id') . ' = :userid')
+				->bind(':userid', $results[0]->user_id, ParameterType::INTEGER);
```

Comment:
` user_id varchar(150) NOT NULL,`
```suggestion
				->bind(':userid', $results[0]->user_id);
```